src/modules/modules.py

```diff
@@ -444,19 +444,29 @@ def __init__(self, pooler, classifier):
 
     def forward(self, sent, mask, idxs=[]):
         """ Assumes batch_size x seq_len x d_emb """
-        emb = self.pooler(sent, mask)
 
+        emb = self.pooler(sent, mask)
         # append any specific token representations, e.g. for WiC task
         ctx_embs = []
-        for idx in [i.long() for i in idxs]:
+        for idx in idxs:
```

Comment:
Add a comment explaining those cases?

Reply:
They're not really cases; for the `torch.gather` we need the `idx` tensor to be three dimensional and this basically checks and modifies the shape.